assets/app/views/_overview-deployment.html

```diff
@@ -49,7 +49,7 @@ <h3 class="h4">
           <span class="deployment-trigger" ng-repeat="cause in rc.causes">
             <span ng-switch="cause.type">
               <span ng-switch-when="ImageChange">
-                a new image <span ng-if="cause.imageTrigger.from">for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
```

Comment:
doesn't seem like we should prevent wrapping, since image trigger names can be long (e.g. "my-namespace-name/image-stream-name:my-tag-name"), but if we do prevent wrapping, make sure truncation looks right, and we add the text as a hover title